index.html

```diff
@@ -1,8 +1,9 @@
 ---
-layout: steps
+layout: default
 ---
 
-<h1>R kernel for Jupyter <img class=logo src="{{site.baseurl}}/images/irkernel-logo.svg"></h1>
+<h1 id="landing-title">R kernel for Jupyter Notebook</h1>
+<img class=logo src="{{site.baseurl}}/images/PoweredbyJupyter-darkborder.svg">
```

Comment:
doesn’t look good just left-aligned below the header.

also i like the idea of having our own logo. where in the brand guidelines does it say we can’t have one using elements from the jupyter logo?